### PR TITLE
Update transforms creator

### PR DESCRIPTION
**Pull request by alicjak1519**

### CODE REVIEW

**Review by wiktorfilipiuk — APPROVED** [review]

**Review by MilenaOlech — APPROVED** [review]

File: datasets/dataset_creator.py [hunk 1]
```diff
@@ -77,3 +77,11 @@ def _get_split_lens(self, dataset: torch.utils.data.ConcatDataset) -> Tuple[int,
 
     def _get_annotations_path(self, video_root: str) -> str:
         return os.path.join(video_root, f'test_{self.classification_mode}.txt')
+
+    @staticmethod
+    def _get_videos_root(data_paths: Union[list, str]) -> list:
```
Comment (majsylw) [inline datasets/dataset_creator.py:82]: as above List[str]

File: train_config.yml [hunk 2]
```diff
@@ -1,4 +1,13 @@
-augmentations:
+general_parameters:
+    data_path: "assets/sanity_check_data"
+    ratio_train_test: 0.8
+    workers: 0
+    gpu: -1
+    path_to_save: "saved_model.pth"
+    seed: 1234
```
Comment (majsylw) [inline train_config.yml:7]: previous experiments were with seed=2021
it ist important becouse the data split depends on that

**Review by majsylw — APPROVED** [review]
